jekyll/_cci2/migrating-from-1-2.md

```diff
@@ -3,332 +3,131 @@ layout: classic-docs
 
-      # The DB is not automatically created in 2.0, so we manually set up a test DB
-      - run:
-          name: Create Test DB
-          command: setup-db-command
-
-      # Again, no inference in 2.0 yet, so nothing to override here, either.
-      # Just use any test commands.
+```
       - run:
-          name: Run Tests
-          command: test-command
+          name: <name>
+          command: <installed-dependency>
 ```
 
-## Caching
-
-### Cache Directories
-
-**1.0**
-
-```YAML
-dependencies:
-  override:
-    - bundle install --path vendor/bundle:
-        timeout: 180
+- Search and replace the `cache_directories:` key:
 
+```
   cache_directories:
     - "vendor/bundle"
 ```
 
-**2.0**
-
-{% raw %}
-```YAML
-version: 2
-jobs:
-  build:
-    working_directory: /root/my-project
-    docker:
-      - image: ruby:2.3
+With the following, nested under `steps:` and customizing for your application as appropriate:
 
-    steps:
-      - checkout
-      - run: bundle install --path vendor/bundle
-
-     # Caching is fully customizable in 2.0
+```
      - save_cache:
-        key: dependency-cache-{{ checksum "Gemfile.lock" }}
+        key: dependency-cache
         paths:
           - vendor/bundle
 ```
-{% endraw %}
 
-Please note that you need to have a restore cache step by yourself in 2.0. There are also lots of cache prefix options available.
+- Add a `restore_cache:` key nested under `steps:`.
 
-Read about them in [Configuration Reference]({{ site.baseurl }}/2.0/configuration-reference/#save_cache).
+- Search and replace `deployment:` with the following, nested under `steps`:
 
-## Deployment
-
-**1.0**
-
-```YAML
-deployment:
-  staging:
-    branch: master
-    heroku:
-      appname: foo-bar-123
+```
+     - deploy:
 ```
 
-**2.0**
-
-Currently, 2.0 doesn't support automatic deployment via integrations (like the above Heroku example).
-
-You can write your own manual `deploy` steps as shown in the [Configuration Reference]({{ site.baseurl }}/2.0/configuration-reference/#deploy).
+**Note:** Currently, 2.0 Beta does not include continuous deployment support. To write your own manual `deploy` steps, refer to the [Configuration Reference]({{ site.baseurl }}/2.0/configuration-reference/#deploy).
 
-If you **add SSH keys** via the CircleCI UI 'Project Settings > SSH Permissions' screen for deployment with 2.0, you must **explicitly add them to your job with: `- add_ssh_keys`** in `.circleci/config.yml`. See [Configuration Reference add SSH keys]({{ site.baseurl }}/2.0/configuration-reference/#add_ssh_keys). The documentation also explains how to activate `ssh-agent` so that you can `ssh-add` your keys for tools that require them to be available via an agent.
+- Validate your YAML again at <http://codebeautify.org/yaml-validator> to check the changes. Fix up any issues and commit the updated `.circle/config.yml` file. CircleCI automatically starts a test run on CircleCI 2.0 with a 2.0 icon on the builds page for that run.
```

Comment:
As above - suggest using local build tool for this: https://circleci.com/docs/2.0/local-jobs/#usage (possibly as well as an online yaml validator)